R/climdex.r

```diff
@@ -560,10 +563,31 @@ climdexInput.raw <- function(tmax=NULL, tmin=NULL, prec=NULL, tmax.dates=NULL, t
   new.date.range <- as.PCICt(paste(as.numeric(format(range(all.dates), "%Y", tz="GMT")), c("01-01", last.day.of.year), sep="-"), cal=cal)
   date.series <- seq(new.date.range[1], new.date.range[2], by="day")
   jdays <- get.jdays.replaced.feb29(get.jdays(date.series))
-  
-  ## Factors for dividing data up
-  date.factors <- list(annual=factor(format(date.series, format="%Y", tz="GMT")), monthly=factor(format(date.series, format="%Y-%m", tz="GMT")))
 
+  # Classify meteorological seasons with associated years
```

Comment:
I see you have a lot of special handling for winter and the year boundary, that's good. 

What happens to the first year in the dataset, does it have  winter measurements taking only two months into account? Is this documented anywhere?

Reply:
I will add a clarification to the documentation. Only seasons where all three constituent months are valid, and the sum of NA days for the season is no greater than six will be valid. This means that a time series starting in January and ending in December will have NA winter seasons at both ends of the series.